samples_project/Assets/SampleViewer/Samples/Viewshed/Scripts/SliderText.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+// Simple text field updater that tracks the value of a slider object
+public class SliderText : MonoBehaviour
+{
+    public string prefix = "";
+    public Slider slider;
+    public TextMeshProUGUI text;
+
+    void Start()
+    {
+        if(slider != null)
+        {
+            slider.onValueChanged.AddListener(delegate { UpdateText(); });
```

Comment:
```suggestion
            slider.onValueChanged.AddListener(UpdateText);
```
?

Reply:
The argument is a UnityAction<float> which takes an unused float parameter, so the suggestion doesn't compile. Modified in latest changes, not sure if it's better than the delegate approach.